### ISSUE TITLE
basisu update to 1.16.4

### ISSUE THREAD

**Issue by sentik** [microsoft/vcpkg#30936]
### Library name

basisu

### New version number

1.16.4

### Other information that may be useful (release notes, etc...)

Now the version of basisu in VCPKG is 1.11, which is very far behind the current version. The current version is 1.16.4 which supports newer and more modern ways of texture compression and optimization for various processors.

https://github.com/BinomialLLC/basis_universal/releases

**Comment by jimwang118** [comment]
Older versions of `basisu `are a fork of the official repository (https://github.com/jherico/basis_universal). This fork has not been maintained since December 2019. In this branch, `basisu` provides external interfaces, including lib, dll and executable programs. But the official library only provides exe, so if we update this port, there are two problems: 
1. The official library only provides exe executable programs, which are generated in tools, and lib and dll are not generated for developers to call. 
2. vcpkg has compilation problems for ports that only generate tools.
```
warning: The folder /include is empty or not present. This indicates the library was not correctly installed.
error: Found 1 post-build check problem(s). To submit these ports to curated catalogs, please first correct the portfile: F:\vcpkg\ports\basisu\portfile.cmake
```
So I will suspend the update of this port and update it after confirming the modification plan of the problem.

**Comment by JonLiu1993** [comment]
Fixed by PR #34408.
